spec/classes/init_spec.rb

```diff
@@ -59,4 +59,20 @@
       end
     end
   end
+
+  context 'on Debian' do
+    describe 'kafka' do
+      context 'offline mirror' do
+        let :params do
+          {
+            offline_mirror_url: 'https://foobar.com/another/kafka'
+          }
+        end
+
+        it {
+          is_expected.to contain_archive('/var/tmp/kafka/kafka_2.11-0.11.0.3.tgz').with(source: 'https://foobar.com/another/kafka')
```

Comment:
The version of Kafka and Scala has changed:
* https://github.com/voxpupuli/puppet-kafka/pull/291